provider_shopper/lib/screens/cart.dart

```diff
@@ -85,11 +85,18 @@ class _CartTotal extends StatelessWidget {
             SizedBox(width: 24),
             TextButton(
               onPressed: () {
-                Scaffold.of(context).showSnackBar(
-                    SnackBar(content: Text('Buying not supported yet.')));
+                //Updated Method of using Snack Bar for more info visit "https://flutter.dev/docs/release/breaking-changes/scaffold-messenger"
+                ScaffoldMessenger.of(context).showSnackBar(SnackBar(
+                  content: const Text('Buying not supported yet.'),
+                  duration: const Duration(seconds: 1),
+                  action: SnackBarAction(
+                    label: ' ',
+                    onPressed: () { },
+                  ),
+                ));
               },
               style: TextButton.styleFrom(primary: Colors.white),
-              child: Text('BUY'),
+              child: Text('BUY',style: TextStyle(fontWeight: FontWeight.bold,color: Colors.black),),
```

Comment:
This needs to be reformatted. Add a comma after `Colors.black` and then run `flutter format` in your IDE or on the command line.